### ISSUE TITLE
Investigate whether tokio-console can be turned on in prod

### ISSUE THREAD

**Issue by umanwizard** [MaterializeInc/materialize#16332]
### Feature request

Someone needs to run some benchmarks to verify that it does not cause any performance issues.

Options for benchmarking are @philip-stoev 's feature benchmark suite and @vmarcos 's scripts that were used for testing columnation work.

**Comment by teskje** [comment]
If we find issues with turning tokio-console on by default, it would be interesting to know if we can run Materialize with tokio-console support by default (i.e. `RUSTFLAGS="--cfg tokio_unstable"` and `--features tokio-console`). Then we would at least be able to switch the console on if needed. Currently we don't have that option, unless we are willing to build new Docker images.

**Comment by pH14** [comment]
This one is unlikely to make the list. I'd love to have it, but it didn't come up as a frequent pain point and afaik the console requires `tokio_unstable`

**Comment by teskje** [comment]
Agreed, observability of tokio tasks has turned out much less of an issue than I'd have expected.
